dipy/reconst/shm.py

```diff
@@ -1060,8 +1060,12 @@ def anisotropic_power(sh_coeffs, norm_factor=0.00001, power=2,
         ap += ap_i
         n_start = n_stop
 
-    # Shift the map to be mostly non-negative:
-    log_ap = np.log(ap) - np.log(norm_factor)
+    # Shift the map to be mostly non-negative,
+    # only applying the log operation to non-zero elements
```

Comment:
Ah! I get it. Well, it is true that the condition is non_zero. It is impossible to have values of ```ap``` that are negative, and we only want to avoid 0s, since those are the ones that throw the log warning.

I agree that with small ap values between 0 and 1, the log will yield a negative value. As I said, it will be handled by the rest of the statement and later code.

I'll push a quick fix to the comment.